internal/controller/postgrescluster/pgbackrest.go

```diff
@@ -1509,8 +1528,16 @@ func (r *Reconciler) reconcilePostgresClusterDataSource(ctx context.Context,
 		return errors.WithStack(err)
 	}
 
+	pgtablespaces := []*corev1.PersistentVolumeClaim{}
+	if util.DefaultMutableFeatureGate.Enabled(util.TablespaceVolumes) {
```

Comment:
@cbandy, what do you feel about removing these gates and leaving `reconcileTablespaceVolumes` as the... gatekeeper?